submissions/promiseChilufya/CSS/CSS/vendor.html

```diff
@@ -131,6 +133,8 @@ <h2>All Vendors</h2>
             </div>
     </main>
     </div>
-</body>
+    <script src="/home/jay/Documents/GitHub/Dufuna-CodeCamp-21/submissions/promiseChilufya/DOM/dom1.js"></script>
```

Comment:
this link will not work immediately on other machines, a better way to put this is ".../DOM/dom1.js. For this line and the next line